Portfolio/index.html

```diff
@@ -4,27 +4,33 @@
     <title>Portfolio</title>
     <meta charset="UTF-8" />
     <link rel="stylesheet" href="index.css" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
   </head>
 
-  <body style="margin: 0; padding: 0">
-    <div top="0">
+  <body>
+    <div class="cont1">
+
       <a href="../index.html">Go Back</a>
-    </div>
+
     <!-- Write your code below for PORTFOLIO and DO NOT TOUCH the code above-->
     <!-- to import image from public folder, do it like this= <img src="../public/css.icon.png" alt="CSS image"> -->

```

Comment:
The 'img' tag should use 'src' attribute instead of 'href'. It should be: <img src='https://cdn.startupsavant.com/images/news/microsoft-chatbot.jpg' alt='chat'/>